src/context/WalletProvider/SelectModal.tsx

```diff
@@ -55,50 +61,103 @@ const WalletSelectItem = ({
 
   const handleConnect = useCallback(() => connect(walletType), [connect, walletType])
 
+  const OptionIcon = option.icon
+  const Icon = useMemo(() => <OptionIcon width='24px' height='auto' />, [OptionIcon])
+
   const isLedgerEnabled = useFeatureFlag('LedgerWallet')
 
   if (walletType === KeyManager.Ledger && !isLedgerEnabled) return null
 
   if (!isSupported) return null
 
-  const isCoinbaseEnabled = getConfig().REACT_APP_FEATURE_COINBASE_WALLET
```

Comment:
do we want to keep this since we are keeping the first class impl now?

Reply:
I'm agnostic though would err on the side of no need, it's been here and stable for quite some time! Those were introduced during development, IMO there's no value into having this flag for Coinbase specifically, if we really wanted to we could flag all wallets for safety to disable them easily but meh